.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,21 +42,16 @@ Documentation change PR (review please): https://github.com/status-im/status.im/
 - network consumption
 
 ### Steps to test
-<!-- (Specify exact steps to test if there are such) -->
+[comment]: #  (Specify exact steps to test if there are such)
 
 - Open Status
 - ...
 - Step 3, etc.
 
-<!-- (PRs will only be accepted if squashed into single commit.) -->
```

Comment:
This is enforced by github machinery.

Reply:
Indeed, this has confused a bunch of people over time. The only thing I'm not sure is that some devs may still prefer to merge via command line, in which case I'm not sure GitHub will reject if the commits aren't squashed. In status-go GitHub happily accepts multiple commits because I've seen this happen a few times.

Reply:
@ilmotta, well, if developer going to use command line, pull request comment won't stop it from happening.